build/Build.proj

```diff
@@ -63,7 +63,8 @@
 
   <Target Name="RestoreCommonVersionInfo" AfterTargets="Build" Condition=" Exists('$(CommonVersionInfoPath).bak') ">
     <Delete Files="$(CommonVersionInfoPath)" Condition=" Exists('$(CommonVersionInfoPath)') " />
-    <Move SourceFiles="$(CommonVersionInfoPath).bak" DestinationFiles="$(CommonVersionInfoPath)" />
+    <Copy SourceFiles="$(CommonVersionInfoPath).bak" DestinationFiles="$(CommonVersionInfoPath)" />
+    <Delete Files="$(CommonVersionInfoPath).bak" Condition=" Exists('$(CommonVersionInfoPath).bak') " />
```

Comment:
xbuild does not have a `Move` task, so `Copy` and `Delete` is the workaround to make it compatible with both xbuild and msbuild. The `Exists` probably isn't necessary, I just used the same logic from the `Delete` a couple lines up.